pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/TabularParticipantParser.java

```diff
@@ -241,7 +241,11 @@ private void addModuleDataToParticipantMap(ModuleExportConfig moduleConfig, Map<
                 if (filterConfig.getChildConfigs() != null) {
                     // if this is a composite question, we need a separate column for each child question
                     for (int childIndex = 0; childIndex < filterConfig.getChildConfigs().size(); childIndex++) {
-                        addSingleResponseToMap(Collections.singletonList(responseValues.get(childIndex)),
+                        List<String> childResponse = new ArrayList<>();
+                        if (childIndex < responseValues.size()) {
```

Comment:
checking childIndex in responseValue
https://broadworkbench.atlassian.net/browse/PEPPER-552